src/components/RecentJobs/index.jsx

```diff
@@ -5,54 +5,49 @@ import SkillsItems from '../SkillsItems';
 import JobModal from '../Modal';
 import modalContext from '../../context/modalContext';
 
-function RecentJobs() {
+function RecentJobs({ toggle }) {
   const { recentJobs } = useContext(JobContext);
-  const [modal, setModal] = useState(false)
-  const [modalID, setID] = useState('')
-  
-  const toggleModal = click => {
-    setID(click.currentTarget.id)
-    setModal(!modal)
-  }
+  const { modal, toggleModal } = useContext(modalContext);
 
   return (
-    <modalContext.Provider value={ {modal: modal, toggleModal: toggleModal, modalID: modalID,} }>
-    <section className='font-sans items-center justify-center mx-auto overflow-hidden px-2.5 w-full md:max-w-screen-md'>
-      <div className='flex flex-row justify-center items-center py-3'>
-        <h3 className='flex justify-start font-bold text-3xl'>Most Recent Jobs</h3>
-        <a className='flex ml-auto font-semibold text-blue-600' href='#'>
-          View all jobs &#8594;
-        </a>
-      </div>
-      <div>
-        <div className='w-full bg-zinc-700'>
-          <div>
-            <div className='flex justify-between px-3 text-white'>
-              <div className='text-left w-[45%]'>Job Title / Company</div>
-              <div className='flex justify-start w-[15%]'>Type</div>
-              <div className='flex justify-start w-[15%]'>Salary</div>
-              <div className='flex justify-start w-[25%]'>Skills</div>
+    <>
+      <section className='font-sans items-center justify-center mx-auto overflow-hidden px-2.5 w-full md:max-w-screen-md' >
+        <div className='flex flex-row justify-center items-center py-3'>
+          <h3 className='flex justify-start font-bold text-3xl'>Most Recent Jobs</h3>
+          <a className='flex ml-auto font-semibold text-blue-600' href='#' onClick={toggle}>
+            View all jobs &#8594;
+          </a>
+        </div>
+        <div>
+          <div className='w-full bg-zinc-700'>
+            <div>
+              <div className='flex justify-between px-3 text-white'>
+                <div className='text-left w-[45%]'>Job Title / Company</div>
+                <div className='flex justify-start w-[15%]'>Type</div>
+                <div className='flex justify-start w-[15%]'>Salary</div>
+                <div className='flex justify-start w-[25%]'>Skills</div>
+              </div>
+            </div>
+            <div>
+              {recentJobs.map((job) => (
+                <JobCard
```

Comment:
I should hav epicked this up on the last story, but can you set each JobCard to have a cursor pointer on hover, just to indicate that they are interactive

Reply:
Done